### PR TITLE
Remove line on the bottom of README.md

### PR DESCRIPTION
**Pull request by KapJI**
I don't think it serves any purpose. Should be cleaner without it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -173,8 +173,6 @@ Code template was mainly taken from [@Ludeeus](https://github.com/ludeeus)'s [in
 
 Under the hood the integration uses [glocaltokens](https://github.com/leikoilja/glocaltokens) python package.
 
----
-
 [buymecoffee]: https://www.buymeacoffee.com/leikoilja
 [buymecoffeebadge]: https://img.shields.io/badge/buy%20me%20a%20coffee-donate-yellow.svg?style=for-the-badge
```
Comment (leikoilja) [inline README.md:177]: Let's also clean the badges we are not using, like these :)

**Review by leikoilja — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (KapJI) [reply README.md:177]: Don't you know to get some coffee? 😀

Comment (leikoilja) [reply README.md:177]: hahha i sure do 😄 
 but c'mon those dont work, there are not that many people willing to contribute to open source in $  😅

Comment (leikoilja) [reply README.md:177]: Plus it's a team work, how are we gonna split all the bucks? 😃

Comment (KapJI) [reply README.md:177]: I was recently surprised how few people are sponsoring HA authors via github sponsors. 🙁

Comment (DurgNomis-drol) [reply README.md]: That does not surprise me unfortunately 😞 I often read some of the issues in home-assistant/core or on the forum and a lot of people often forget that people do the work for free (excluding Nabu Casa).

Comment (leikoilja) [reply README.md]: yeeeeh, unfortunatelly that's the world we are living in 😄  
we can ofc think about starting to monetarizing the integration once we have a solid base of users 😅  jk

Comment (DurgNomis-drol) [reply README.md]: Like a 1 dollar per device? 🤣

**Pull request merged**